test/app-luatest/gh_4317_console_backslash_test.lua

```diff
@@ -0,0 +1,63 @@
+local t = require('luatest')
```

Comment:
Can't we arrange the code of the test similarly to `test/app-luatest/gh_7031_configure_eos_in_lua_console_test.lua`? There are some gains: request/response locality, no global socket path, no `listen()` in the main instance that runs all the tests (when `luatest` is called directly), no repeating 'readline on CentOS 7' copy-paste.

Maybe I already asked. Excuse me if so.

It doesn't look as a big effort if there are no problems on the road. If there are some, I would like to know: at least I can adjust the `interative_tarantool.lua` helper to reduce a boilerplate in a future.

Reply:
I've tried this, but `interactive_tarantool` has some readline hacks that were not working in my case. I dm'ed you the details.

Reply:
I'm okay to postpone it.